docs/assets/scripts/extra_script.js

```diff
@@ -0,0 +1,13 @@
+let logo_title = document.getElementsByClassName('md-header__topic')[0]
+logo_title.className = 'md-header__button md-header__topic'
+
+logo_title.onclick = function() {
+    location.href = document.location.origin;
+  };
+logo_title.onmouseover = function(){
```

Comment:
cant you do this with css? lol

Reply:
oh yeah was in my js file forgot about it lmao